liblwgeom/lwgeom_geos.c

```diff
@@ -549,18 +586,18 @@ lwgeom_intersection(const LWGEOM *geom1, const LWGEOM *geom2)
 
 	LWDEBUG(3, "intersection() START");
 
-	g1 = LWGEOM2GEOS(geom1, 0);
+	g1 = LWGEOM2GEOS_PREC(geom1, LW_FALSE, op_precision);
```

Comment:
We still want to be able to work with floating grid, how can it be done ?

Reply:
@strk By passing `NULL` for the `op_precision` argument